bullet-featherstone/src/Base.cc

```diff
@@ -42,6 +42,13 @@ WorldInfo::WorldInfo(std::string name_)
   // Needed for force-torque sensor
   this->world->getSolverInfo().m_jointFeedbackInJointFrame = true;
   this->world->getSolverInfo().m_jointFeedbackInWorldSpace = false;
+
+  // By default a large impulse is applied when collisions penetrate
+  // which causes unstable behavior. Bullet featherstone does not support
+  // configuring split impulse and penetration threshold parameters. Instead the
+  // penentration impulse depends on the erp2 parameter so set to a small value
+  // (default is 0.2).
+  this->world->getSolverInfo().m_erp2 = 0.002;
```

Comment:
windows compiler warning here

* https://build.osrfoundation.org/job/gz_physics-pr-win/98/msbuild/

~~~
'=': truncation from 'double' to 'btScalar'
~~~

Reply:
fixed in https://github.com/gazebosim/gz-physics/pull/602/commits/92e02c3901e084b9d2d5324804f78173b95e8b36